llvm-project/build.sh

```diff
@@ -1,3 +1,6 @@
 #!/usr/bin/env sh
 docker build --tag clangbuiltlinux/llvm-project:latest .
-docker cp llvm-project:/usr/local/bin/clang-14 clang
+#docker create --name llvm-project clangbuiltlinux/llvm-project:latest
+#docker cp llvm-project:/usr/local/bin/clang-14 clang
+dot -Tsvg stage1.dot > stage1.svg
+dot -Tsvg stage2.dot > stage2.svg
```

Comment:
Not sure that these belong here.  Perhaps have a separate Makefile for the doc images?

Reply:
done PTAL